apps/editors/views.py

```diff
@@ -144,8 +149,10 @@ def home(request):
                  ('med', _('Passable (5 to 10 days)')),
                  ('old', _('Overdue (Over 10 days)')))
 
-    progress, percentage = _editor_progress()
-    unlisted_progress, unlisted_percentage = _editor_progress(unlisted=True)
+    limited_reviewer = is_limited_reviewer(request)
+    progress, percentage = _editor_progress(limited_reviewer=limited_reviewer)
+    unlisted_progress, unlisted_percentage = _editor_progress(unlisted=True,
+                                             limited_reviewer=limited_reviewer)
```

Comment:
Flake8 wants you to be consistent about indents; you need to choose either a hanging indent or a vertical indent. This will make the error go away:
````
unlisted_progress, unlisted_percentage = _editor_progress(
    unlisted=True, limited_reviewer=limited_reviewer)
````